src/main.tsx

```diff
@@ -1,8 +1,16 @@
 import "./css/style.css";
+import "../node_modules/identity-style-guide/dist/assets/scss/_styles.scss";
+
 import { render } from "preact";
 import { Link } from "./router";
 import { Routes } from "./routes";
 
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import * as USWDS from "../node_modules/uswds/dist/js/uswds";
```

Comment:
This disable comment did not get rid of the error I wanted it to :/

```
src/main.tsx:9:24 - error TS7016: Could not find a declaration file for module '../node_modules/uswds/dist/js/uswds'. 'identity-reporting/node_modules/uswds/dist/js/uswds.js' implicitly has an 'any' type.

9 import * as USWDS from "../node_modules/uswds/dist/js/uswds";
```

Reply:
there's probably a better place to put this import, but adding the `<script>` in `index.html` didn't seem to work, might need to try again